### PR TITLE
web: Fix incorrect title on scrolling note

### PR DESCRIPTION
**Pull request by Parthiv-M**

### CODE REVIEW

File: apps/web/src/components/editor/toolbar.jsx [hunk 1]
```diff
@@ -92,6 +92,10 @@ function Toolbar() {
     };
   }, [isTitleVisible]);
 
+  useEffect(() => {
+    document.querySelector("#onScrollTitle").defaultValue = title;
```
Comment (thecodrr) [inline apps/web/src/components/editor/toolbar.jsx:96]: Use a ref instead of `document.querySelector`.

**Review by thecodrr — CHANGES_REQUESTED** [review]

**Comment by thecodrr** [comment]
@Parthiv-M thanks for fixing this!

**Comment by Parthiv-M** [comment]
I've made the change

File: apps/web/src/components/editor/toolbar.jsx [hunk 2]
```diff
@@ -93,7 +94,7 @@ function Toolbar() {
   }, [isTitleVisible]);
 
   useEffect(() => {
-    document.querySelector("#onScrollTitle").defaultValue = title;
+    onScrollTitleRef.current.defaultValue = title;
```
Comment (thecodrr) [inline apps/web/src/components/editor/toolbar.jsx:97]:
```suggestion
    if (onScrollTitleRef.current)
        onScrollTitleRef.current.defaultValue = title;
```

**Review by thecodrr — APPROVED** [review]

**Pull request merged**